showCount.test.js

```diff
@@ -0,0 +1,13 @@
+import getComments from './modules/getComments';
+import showLikes from './modules/showLikes';
+
+it('should count the number of shows', async () => {
+  const likes = await showLikes();
+  const count = likes.length;
+  expect(count).toEqual(15);
+});
+it('should count the number of comments for the show, Person of interest', async () => {
+  const comments = await getComments(2);
+  const count = comments.length;
+  expect(count).toEqual(9);
+});
```

Comment:
- [ ] It will be great to ensure that the test cases are not failing. It seems your tests are failing kindly check below ⬇️ 
![javascriptTest](https://github.com/ctokoli/JavaScript-Group-Capstone-Project/assets/32361963/3e9d9c20-97be-455f-99ea-1088b15e53a7)

- [ ] Kindly also Make sure that you have tests for all edge cases. Example: What if there are 0 items on the page (counter function should return 0)